### ISSUE TITLE
A11y_Model Builder_Object Detection_Environment_Keyboard: "Which (GPU) are Compatible" link is not accessible with keyboard present under "Local(GPU)" Screen.

### ISSUE THREAD

**Issue by Harika1702** [dotnet/machinelearning-modelbuilder#2675]
### GitHub Tags
#WCAG2.1.1; #A11ySev2; #A11yTCS; #A11yWCAG2.1; #DesktopApp; #Win11; #Visual Studio Client; #SH_ModelBuilder_Win32_May2023; #FTP; #Win32; #Benchmark; #Keyboard; #Tabstops; #A11yAuto

### Environment Details
Application Name: Model Builder
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.6.1
Microsoft Windows 11 Enterprise Build 22624.1755

### Repro Steps
1. Launch the Visual Studio with at least one of the .NET workloads.
2. Tab till enable ML.NET Model Builder (Tools -> Options -> Environment -> Preview Features) and select it.
3. Create a new .NET project or open an existing .NET project.
4. Open context menu (Right-click) of the project in the solution explorer.
5. Tab till Add and then tab till Machine Learning Model and press enter.
6. Model Builder dialog will open. Tab till Scenario tab.
7. Select Scenario screen will appear. Tab till "Object Detection" tile and press enter.
8. Tab till "Local (GPU)" and press enter.
9. Now tab till "Which GPU) are Compatible."
10. Now verify that "Which (GPU) are Compatible" link is not accessible with keyboard present under "Local (GPU)" Screen.

### Actual Result
"Which (GPU) are Compatible" link is not accessible with keyboard present under "Local (GPU)" Screen.
**Note:
We are facing similar issue throughout the application.**

### Expected Result
"Which (GPU) are Compatible" link should be accessible with keyboard present under "Local (GPU)" Screen.

### User Impact
Keyboard users will not be able to navigate/ access the control if it is not accessible with keyboard.

### Attachment
<img width="960" alt="Bug_Keyboard" src="https://github.com/dotnet/machinelearning-modelbuilder/assets/100270074/431915ba-e1c1-4b17-bc49-7e4bfc975094">

![Bug_Keybord_](https://github.com/dotnet/machinelearning-modelbuilder/assets/100270074/7667160b-b89e-4922-a55c-51fd6c9cb48b)

**Comment by zewditu** [comment]
@Harika1702 , ready to validate by Model Builder version 16.17.0.2332602

**Comment by Yogi2204** [comment]
#Regressed:06-29-23; #Closed;
Verified the issue in below environment:
Visual Studio : Microsoft Visual Studio Enterprise 2022 (64-bit) - Current Version 17.6.3
Model Builder : Version 17.17.0.2332602

The issue is fixed, hence closing the bug. Please find attachment below for reference. Thanks!
[Recording-20230629_172029.webm](https://github.com/dotnet/machinelearning-modelbuilder/assets/130456851/5957cb73-8ce3-460f-a27a-6963deabe97b)
